docs/user/simultaneous_use.rst

```diff
@@ -0,0 +1,33 @@
+Simultaneous session limits (Simultaneous-Use)
+==============================================
+
+``Simultaneous-Use`` limits how many sessions a user can have active at
+the same time.
+
+During authorization, OpenWISP RADIUS counts the user's active sessions
+and compares them with the limit set in their RADIUS group. If the number
+of active sessions is **equal to or greater than** the limit, access is
+denied with:
+
+.. code-block:: text
+
+    You are already logged in - access denied
+
+.. note::
+
+    A value of ``0`` or negative means **no limit**.
```

Comment:
In this case wouldn't it be better to just delete the check?

Reply:
Yes, it'd be. FreeRADIUS deals with zero or negative values in this way. 

I will remove this note.